README.md

```diff
@@ -19,7 +19,7 @@ Python 3.5 and some select features from later versions). The following core
 datatypes are provided: `str`(including basic Unicode support), `bytes`,
 `bytearray`, `tuple`, `list`, `dict`, `set`, `frozenset`, `array.array`,
 `collections.namedtuple`, classes and instances. Builtin modules include
-`os`, `sys`, `time`, `re`, and `struct`, etc. Select ports have support for
+`os`, `sys`, `time`, `re`, and `struct`, etc. Selected ports have support for
```

Comment:
Maybe more clear to say "Some" or "Certain"?

"Select ports" makes more sense to me than "Selected ports" though.

Reply:
Hi! Thanks for the feedback. I agree that `Some` or `Certain` could work well here. `Some` would be a better choice.